sources/kube_factory.go

```diff
@@ -31,11 +32,13 @@ import (
 )
 
 const (
-	defaultApiVersion     = "v1beta3"
-	defaultInsecure       = false
-	defaultKubeletPort    = 10255
-	defaultKubeConfigFile = "/etc/kubernetes/kubeconfig/kubeconfig"
-	defaultKubeletHttps   = false
+	defaultApiVersion         = "v1beta3"
+	defaultInsecure           = false
+	defaultKubeletPort        = 10255
+	defaultKubeConfigFile     = "/etc/kubernetes/kubeconfig/kubeconfig"
+	defaultKubeletHttps       = false
+	defaultUseServiceAccount  = false
+	defaultServiceAccountFile = "/var/run/secrets/kubernetes.io/serviceaccount/token"
```

Comment:
I'm working on it for OpenShift Origin docs & will either link to it from here or add it here too. There are some problems to work through as OpenShift is locked down right now so accessing /stats endpoint on the kubelet isn't possible... working on that too,